pkg/handlers/v1/nexposevulnnotifier.go

```diff
@@ -0,0 +1,59 @@
+package v1
+
+import (
+	"context"
+
+	"github.com/asecurityteam/nexpose-vuln-notifier/pkg/domain"
+	"github.com/asecurityteam/nexpose-vuln-notifier/pkg/logs"
+)
+
+// ScanInfo represents the incoming payload
+type ScanInfo struct {
+	// ScanID is the ID of the completed scan whose vulnerabilities we'll process
+	ScanID string `json:"scan_id"`
```

Comment:
It doesn't look like `ScanID` ever gets used, and it's not required in the payload. Is this intended to be used in the future?

Reply:
I'm not sure if it will be used in the future. This matches the payload that Splunk will send us, so I added it here. Does having it here make it confusing? Should I remove it?
cc. @faldridge do you think we'll use ScanID in the future?

Reply:
In that case I think the comments are a little misleading. I think it'd be fine to leave it in the API spec, but drop it here until it's required, or else document that it's unused but planned for future work.

Reply:
Okay